SpiceSharp/Components/RLC/RES/FrequencyBehavior.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
+using System.Numerics;
+using SpiceSharp.Sparse;
+using SpiceSharp.Attributes;
+using SpiceSharp.Simulations;
+using SpiceSharp.Behaviors;
+
+namespace SpiceSharp.Components.ResistorBehaviors
+{
+    /// <summary>
+    /// AC behavior for <see cref="Resistor"/>
+    /// </summary>
+    public class FrequencyBehavior : Behaviors.FrequencyBehavior, IConnectedBehavior
+    {
+        /// <summary>
+        /// Parameters
+        /// </summary>
+        [PropertyName("v"), PropertyInfo("Voltage")]
+        public Complex GetVoltage(ComplexState state)
+        {
+			if (state == null)
+				throw new ArgumentNullException(nameof(state));
+
+            return state.Solution[posNode] - state.Solution[negNode];
+        }
+        [PropertyName("i"), PropertyInfo("Current")]
+        public Complex GetCurrent(ComplexState state)
+        {
+			if (state == null)
+				throw new ArgumentNullException(nameof(state));
+
+            var voltage = state.Solution[posNode] - state.Solution[negNode];
+            return voltage * load.Conductance;
+        }
+        [PropertyName("p"), PropertyInfo("Power")]
+        public Complex GetPower(ComplexState state)
+        {
+			if (state == null)
+				throw new ArgumentNullException(nameof(state));
+
+            var voltage = state.Solution[posNode] - state.Solution[negNode];
+            return voltage * Complex.Conjugate(voltage) * load.Conductance;
+        }
+
+        /// <summary>
+        /// Necessary behaviors
+        /// </summary>
+        LoadBehavior load;
+
+        /// <summary>
+        /// Nodes
+        /// </summary>
+        int posNode, negNode;
+        protected Element<Complex> PosPosPtr { get; private set; }
+        protected Element<Complex> NegNegPtr { get; private set; }
+        protected Element<Complex> PosNegPtr { get; private set; }
+        protected Element<Complex> NegPosPtr { get; private set; }
+
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="name">Name</param>
+        public FrequencyBehavior(Identifier name) : base(name) { }
+
+        /// <summary>
+        /// Setup the behavior
+        /// </summary>
+        /// <param name="provider">Data provider</param>
+        public override void Setup(SetupDataProvider provider)
+        {
+			if (provider == null)
+				throw new ArgumentNullException(nameof(provider));
+
+            // Get behaviors
+            load = provider.GetBehavior<LoadBehavior>(0);
+        }
+        
+        /// <summary>
+        /// Get matrix pointers
+        /// </summary>
+        /// <param name="matrix">Matrix</param>
+        public override void GetMatrixPointers(Matrix<Complex> matrix)
+        {
+			if (matrix == null)
```

Comment:
@svenboulanger Spacing issue. General comment

Reply:
I can't see these spacing issues in Visual Studio... I am reformatting the code to be sure.